test/core/primitives/ticket/ticket_test.cpp

```diff
@@ -33,26 +54,57 @@ TEST_F(TicketTest, SameTicketEqualSuccess) {
 TEST_F(TicketTest, DIfferentTicketEqualFailure) {
   auto &&t1 = generator.makeTicket();
   auto &&t2 = generator.makeTicket();
-  ASSERT_FALSE(t1 == t2);
+  ASSERT_FALSE(t1 == t2) << print(t1) << print(t2);
+  ;
 }
 
 /**
  * @given two different tickets t1, t2
  * @when compare them using operator< (`less`)
  * @then one and only one of t1 < t2 and t2 < t1 is true
  */
-TEST_F(TicketTest, TwoTicketsLessSuccess) {
+TEST_F(TicketTest, EnsureLessIsAntisimmetricSuccess) {
   auto &&t1 = generator.makeTicket();
   auto &&t2 = generator.makeTicket();
-  ASSERT_TRUE( (t1 < t2) != (t2 < t1));
+  ASSERT_TRUE((t1 < t2) != (t2 < t1)) << print(t1) << print(t2);
+}
+
+/**
+ * @given a random generated ticket
+ * @when compare it to itself using operator< (`less`)
+ * @then the result is false
+ */
+TEST_F(TicketTest, EnsureLessIsAntireflexiveSuccess) {
+  auto &&t = generator.makeTicket();
+  ASSERT_FALSE(t < t) << print(t);
+}
+
+/**
+ * @given two tickets, for which it is known that one is less than other if
+ * compared alphabetically
+ * @when compare them using operator<(`less`)
+ * @then the result of comparison is true
+ */
+TEST_F(TicketTest, TicketsCompareLessSuccess) {
+  ASSERT_TRUE(ticket_small < ticket_big);
 }
 
 /**
  * @given a ticket and a round value
- * @when drawRamdomness method is applied
+ * @when `drawRamdomness` method is applied
  * @then result is success
  */
 TEST_F(TicketTest, DrawRandomnessSuccess) {
   auto &&t1 = generator.makeTicket();
-  EXPECT_OUTCOME_TRUE_1(fc::primitives::ticket::drawRandomness(t1, 1u));
+  EXPECT_OUTCOME_TRUE_1(drawRandomness(t1, 1u));
+}
+
+/**
+ * @given predefined ticket, round value and lotus-drawn randomness
+ * @when call drawRandomness using given ticket and round value
+ * @then resulting randomness value is equal to lotus-calculated value
+ */
+TEST_F(TicketTest, DrawRandomnessLotusSuccess) {
+  EXPECT_OUTCOME_TRUE(randomness, drawRandomness(ticket_small, round_value));
```

Comment:
`EXPECT_OUTCOME_EQ(drawRandomness(ticket_small, round_value), randomness_value)`

Reply:
I don't think it is better.